### ISSUE TITLE
Webpack override (partial) config

### ISSUE THREAD

**Issue by lancetharper** [rei/vunit#14]
I've added a `webpack.overrides.conf.js` file for [febs](https://github.com/rei/febs#adding-webpack-loaders) to add a resolve alias but it looks like vunit doesn't use that file and only has an option to provide a full webpack config.

Is there some way to add just a partial config that would merge with the default one? Or do I have to get the full config from febs somehow to pass to this?

**Comment by peripateticus** [comment]
Yes, currently `vunit` doesn't consume the config that `febs` is consuming but I'm going to be doing some work to integrate the configs so that `vunit` will use the same config that `febs` is using. Thanks for filing the issue @lancetharper!
